src/ts/InternalBitmovinYospacePlayer.ts

```diff
@@ -911,28 +913,56 @@ export class InternalBitmovinYospacePlayer implements BitmovinYospacePlayerAPI {
 
   private mapEmsgToId3Tags(event: MetadataEvent): TimedMetadata {
     const metadata = event.metadata as any;
-    const yospaceMetadataObject: any = {
-      startTime: metadata.presentationTime ? metadata.presentationTime : this.player.getCurrentTime(),
-    };
+    const startTime = metadata.presentationTime ? metadata.presentationTime : (this.player.getCurrentTime() as any);
```

Comment:
1. Please remove the `as any`, shouldn't be necessary.
2. I just realized that the original code is actually buggy, because it doesn't do what you'd expect if `metadata.presentationTime` is `0`. Let's fix this right away please. The below suggestion should be safer to use, and is type safe because Typescript can now infere that `startTime` must be a `number`.

```suggestion
    const startTime = !isNaN(metadata.presentationTime) ? Number(metadata.presentationTime) : this.player.getCurrentTime();
```